code/testapp/src/main/java/com/adobe/marketing/optimizeapp/OffersScreen.kt

```diff
@@ -113,6 +119,8 @@ fun OffersView(viewModel: MainViewModel) {
                         }
                     }
                 })
+            }.also {
+                displayPropositionsHandler(viewModel.optimizePropositionStateMap.values.toList())
```

Comment:
Please remove existing `displayHandler`, else you will get two calls for very offer

Reply:
commented this @spoorthipujariadobe